sources/packages/api/src/route-controllers/institution/models/institution.dto.ts

```diff
@@ -74,32 +71,57 @@ export class InstitutionDto extends PartialType(CreateInstitutionDto) {
   @IsOptional()
   institutionTypeName: string;
 }
+export class InstitutionAddress {
+  @IsNotEmpty()
+  addressLine1: string;
+  @Allow()
+  addressLine2: string;
+  @IsNotEmpty()
+  city: string;
+  @IsNotEmpty()
+  provinceState: string;
+  @IsNotEmpty()
+  country: string;
+  @IsNotEmpty()
+  postalCode: string;
+}
 
-export class AESTInstitutionDetailDto {
-  legalOperatingName: string;
+export class InstitutionContactDTO {
+  @IsNotEmpty()
+  primaryContactEmail: string;
+  @IsNotEmpty()
+  primaryContactFirstName: string;
+  @IsNotEmpty()
+  primaryContactLastName: string;
+  @IsNotEmpty()
+  primaryContactPhone: string;
+  @ValidateNested()
+  @Type(() => InstitutionAddress)
+  mailingAddress: InstitutionAddress;
+}
+
+export class InstitutionProfileDTO extends InstitutionContactDTO {
+  @IsNotEmpty()
   operatingName: string;
+  @IsNotEmpty()
   primaryPhone: string;
+  @IsNotEmpty()
   primaryEmail: string;
+  @IsNotEmpty()
   website: string;
+  @IsNotEmpty()
   regulatingBody: string;
-  institutionTypeName: string;
-  formattedEstablishedDate: string;
-  primaryContactEmail: string;
-  primaryContactFirstName: string;
-  primaryContactLastName: string;
-  primaryContactPhone: string;
-  legalAuthorityEmail: string;
-  legalAuthorityFirstName: string;
-  legalAuthorityLastName: string;
-  legalAuthorityPhone: string;
-  address: {
-    addressLine1: string;
-    addressLine2: string;
-    city: string;
-    country: string;
-    provinceState: string;
-    postalCode: string;
-  };
+  @IsNotEmpty()
+  establishedDate: Date;
+  @IsNotEmpty()
```

Comment:
There are specific class validators for numbers, like the below one to validate an ID
```
@IsNotEmpty()
@IsInt()
@Min(1)
```